lib/Parser.js

```diff
@@ -714,34 +819,37 @@ class Parser extends Tapable {
 
 	prewalkExportDefaultDeclaration(statement) {
 		if(/Declaration$/.test(statement.declaration.type)) {
-			const pos = this.scope.definitions.length;
+			const originalDefinitions = this.scope.definitions;
+			const tracker = new TrackingSet(this.scope.definitions);
+			this.scope.definitions = tracker;
 			this.prewalkStatement(statement.declaration);
-			const newDefs = this.scope.definitions.slice(pos);
+			const newDefs = Array.from(tracker.getAddedItems());
+			this.scope.definitions = originalDefinitions;
 			for(let index = 0, len = newDefs.length; index < len; index++) {
 				const def = newDefs[index];
```

Comment:
This can be a `for (const def of tracker.getAddedItems())` but I don’t know what performance impact it’d have.